modules/styling/lib/cs.ts

```diff
@@ -298,18 +299,23 @@ export interface CSProps {
    * returned by {@link createStyles}, or the result of {@link createVars} and
    * {@link createModifiers}. If you're extending a component already using `cs`, you can merge that
    * prop in as well. Any style that is passed to the `cs` prop will override style props. If you
-   * wish to have styles that are overridden by style props, the `css` prop, or styles added via
-   * the `styled` API, use {@link mergeStyles} wherever `elemProps` is used.
+   * wish to have styles that are overridden by the `css` prop, or styles added via the `styled`
+   * API, use {@link handleCsProp} wherever `elemProps` is used. If your component needs to also
+   * handle style props, use {@link mergeStyles} instead.
    *
    *
    * ```tsx
+   * import {handleCsProp} from '@workday/canvas-kit-styling';
    * import {mergeStyles} from '@workday/canvas-kit-react/layout';
    *
    * // ...
    *
+   * // `handleCsProp` handles compat mode with Emotion's runtime APIs. `mergeStyles` has the same
+   * // function signature, but adds support for style props.
+   *
    * return (
    *   <Element
-   *     {...mergeStyles(elemProps, [
+   *     {...handleCsProp(elemProps, [
```

Comment:
We're promoting `handleCsProp` over `mergeStyles`. `mergeStyles` now handles style props which we don't want to promote. We'll use it internally for backwards compatibility.